admin/cluster_setup.py

```diff
@@ -11,23 +11,159 @@
 from pipes import quote as shell_quote
 
 from eliot import FileDestination, add_destination, write_failure
+from pyrsistent import pvector
+from txeffect import perform
 
-from twisted.internet.defer import inlineCallbacks
-from twisted.python.usage import UsageError
+from twisted.internet.defer import DeferredList, inlineCallbacks
 from twisted.python.filepath import FilePath
+from twisted.python.usage import UsageError
 
 from .acceptance import (
+    CLOUD_PROVIDERS,
     ClusterIdentity,
     CommonOptions,
+    LibcloudRunner as OldLibcloudRunner,
     capture_journal,
     capture_upstart,
     eliot_output,
+    get_default_volume_size,
     get_trial_environment,
+    save_backend_configuration,
 )
 
 from flocker.apiclient import FlockerClient
 from flocker.common import loop_until
 from flocker.control.httpapi import REST_API_PORT
+from flocker.provision._ca import Certificates
+from flocker.provision._common import Cluster
+from flocker.provision._install import configure_control_node
+from flocker.provision._ssh._conch import make_dispatcher
+
+
+class LibcloudRunner(OldLibcloudRunner):
+    """
+    An alternative approach to setting up a cluster using
+    a libcloud-compatible provisioner.
+    """
+    def _setup_control_node(self, reactor, node, index):
+        print "Selecting node {} for control service".format(node.name)
+        certificates = Certificates.generate(
+            self.cert_path,
+            node.address,
+            0,
```

Comment:
I think self.cert_path, node.address and particularly '0' deserve the <argument>= in front of them. Someone reading this would have no idea what 0 is, specifically